src/lang/de.ts

```diff
@@ -0,0 +1,7 @@
+const dict = {
+  today: "Meute",
```

Comment:
This should be "Heute". I'd happily provide the rest of the translations whenever this gets merged or you develop it further. (And thanks for adding multilingual support.)

Reply:
Thanks, fixed. I merged master back into this branch and set it as ready for review.